src/Molecules/TableList/TableList.jsx

```diff
@@ -43,64 +44,58 @@ export const TableList = ({
   way,
   Cell = ({ children }) => <TableCell children={children || '-'} />,
   Column = BaseColumn,
-  ColumnProps = {},
-  emptyPlaceholder = null,
   ...props
-}) => {
-  return data.length === 0 && !isCurrentlyFiltered(filters) ? emptyPlaceholder :
-    <Wrapper>
-      <Table {...props}>
-        <Head>
-          <TableRow>
-            { columns.map(({ name, translationKey, ...column }) => (
-              <Column
-                key={name}
-                onSort={onSort}
-                sort={sort === name && way}
-                filter={filters[name]}
-                onFilter={value => {
-                  onFilter({ ...filters, [name]: value })
-                }}
-                onClear={name => onClear({ ...filters, [name]: null })}
-                name={name}
-                translationKey={translationKey}
-                {...column}
-                {...ColumnProps}
-              >
-                <Trans transKey={translationKey} />
-              </Column>
-            )) }
-          </TableRow>
-        </Head>
-        <TableBody>
-          { data.length === 0 ? (
-            <Row hover={false}>
-              <TableCell colSpan={columns.length}>
-                <NoResults>
-                  <Trans>global.no_results</Trans>
-                </NoResults>
-              </TableCell>
+}) => (
+  <Wrapper>
+    <Table {...props}>
+      <Head>
+        <TableRow>
+          { columns.map(({ name, translationKey, ...column }) => (
+            <Column
+              key={name}
+              onSort={onSort}
+              sort={sort === name && way}
+              filter={filters[name]}
+              onFilter={value => onFilter({ ...filters, [name]: value })}
+              onClear={defaultValue => onFilter({ ...filters, [name]: defaultValue })}
+              name={name}
+              translationKey={translationKey}
+              {...column}
+            >
+              <Trans transKey={translationKey} />
+            </Column>
+          )) }
+        </TableRow>
+      </Head>
+      <TableBody>
+        { data.length === 0 ? (
+          <Row hover={false}>
+            <TableCell colSpan={columns.length}>
+              <NoResults>
+                <Trans>global.no_results</Trans>
+              </NoResults>
+            </TableCell>
+          </Row>
+        ) :
+          data.map(item => (
+            <Row
+              key={item.id}
+              id={item.id}
+              selected={selected === item.id}
+              cols={columns.length}
+              onSelect={onSelect}
+              details={<Details { ...item }/>}
+            >
+              { columns.map(({ name }) => (
+                <Cell key={name} name={ name } item={ item }>
+                  {item[name]}
+                </Cell>
+              )) }

```

Comment:
Not for this PR but just a reminder that we'll probably have to do something perfomancewise about this nested `array.map`